modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -1644,6 +1644,10 @@
 		elseif _ACTION >= "vs2019" and cfg.toolset and cfg.toolset == "clang" then
 			local value = iif(cfg.unsignedchar, "On", "Off")
 			table.insert(opts, p.tools.msc.shared.unsignedchar[value])
+			-- <OpenMPSupport>true</OpenMPSupport> is unfortunately ignored with clang toolset
+			if cfg.openmp == "On" then
+				table.insert(opts, 1, '-openmp')
+			end
```

Comment:
Could this use the same as the above two lines and use the MSC toolset value? It looks like Clang is fine with `/J`, so it might be fine with `/openmp` and `/openmp-`? It looks like the "real" Clang flag is `-fopenmp` based on [this](https://github.com/premake/premake-core/blob/f4e88bbd925433181f52ec31a2a96c95ddfef238/src/tools/gcc.lua#L70-L72).

Thoughts?

Reply:
Changed to `'/openmp'` which works too. (`-fopenmp` doesn't work IIRC, msvc translates option to clang).

Reply:
Sorry, what I meant is could this be written like the above lines which don't duplicate the flag but instead get it out of MSC? So, instead of:
```lua
table.insert(opts, 1, '/openmp')
```
it would be:
```lua
table.insert(opts, 1, p.tools.msc.shared.openmp[cfg.openmp])
```

I'm just not sure if it would accept `/openmp-` (from `openmp "Off"`) as a valid flag?

Reply:
@Jarod42 to clarify, that's not me requesting this change but asking if you think it's worth it?